alicloud-visualbasic/${PROJECT}.vbproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
-    <RootNamespace></RootNamespace>
+    <RootNamespace>PulumiVbTemplate</RootNamespace>
```

Comment:
What's the benefit of changing this? Is that specific to VB? An empty namespace feels more natural to me than `PulumiVbTemplate` that users will need to change 100% of the time.